src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,128 @@
+import mate.jdbc.util.ConnectionUtil;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final Long FIRST_ID = 1L;
+    private static final Long SECOND_ID = 2L;
+    private static final Long THIRD_ID = 3L;
+    private static final int FOURTH_DRIVER = 3;
+    private static final int FIRST_DRIVER = 1;
+    private static final int FIRST_MANUFACTURER = 0;
+    private static final int SECOND_MANUFACTURER = 1;
+    private static final String SEP = System.lineSeparator();
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        List<Manufacturer> manufacturers = testManufacturerService();
+        List<Driver> drivers = testDriverService();
+        testCarService(manufacturers, drivers);
+        // UNCOMMENT METHOD BELOW WITH CAUTION
+        //clearAll();
+    }
+
+    private static void testCarService(List<Manufacturer> manufacturers, List<Driver> drivers) {
+        System.out.println(SEP + "CAR TEST SERVICE" + SEP);
+        Car amarok = carService.create(
+                new Car("Amarok",
+                        manufacturers.get(FIRST_MANUFACTURER),
+                        drivers));
+        Car santaFe = carService.create(
+                new Car("Santa Fe",
+                        manufacturers.get(SECOND_MANUFACTURER),
+                        drivers.stream().limit(FIRST_DRIVER).collect(Collectors.toList())));
+        System.out.println(carService.get(SECOND_ID));
+        carService.addDriverToCar(drivers.get(FOURTH_DRIVER), santaFe);
+        carService.getAll().forEach(System.out::println);
+        carService.getAllByDriver(FIRST_ID).forEach(System.out::println);
+        System.out.println(carService.get(SECOND_ID));
+        carService.removeDriverFromCar(drivers.get(FOURTH_DRIVER), santaFe);
+        System.out.println(carService.get(SECOND_ID));
+        carService.delete(SECOND_ID);
+        carService.getAll().forEach(System.out::println);
+    }
+
+    private static List<Driver> testDriverService() {
+        System.out.println(SEP + "DRIVER TEST SERVICE" + SEP);
+        List<Driver> drivers = new ArrayList<>();
+        Driver bob = driverService.create(
+                new Driver("Bob", "NY12345678"));
+        drivers.add(bob);
+        Driver john = driverService.create(
+                new Driver("John", "FL12345678"));
+        drivers.add(john);
+        Driver alice = driverService.create(
+                new Driver("Alice", "NE12345678"));
+        System.out.println(driverService.get(SECOND_ID));
```

Comment:
```suggestion
        System.out.println(driverService.get(john.getId()));
```
same everywhere
please don't use hardcoded ids